doc/interoperability/pandas.rst

```diff
@@ -0,0 +1,288 @@
+
+.. _pandas:
+
+=======================
+Pandas Interoperability
+=======================
+
+This part of the User Guide aims to highlight some of the particularities that
+arise when using Pandas datastructures as input for Scikit-learn.
+
+The basics of using Pandas and Scikit-learn
+==================================================================
+
+Scikit-learn supports the use of
+`Pandas DataFrames <http://pandas.pydata.org/pandas-docs/stable/>`__
+implicitly. However, this implicit support has its conditions and potential
+pitfalls and some (not all) of these will be outlined below.
```

Comment:
```suggestion
pitfalls, some of which are outlined below.
```

Your text is generally very verbose, but I won't give it a full copy edit. Perhaps you can look over it for sentences you can say in fewer, simple words and paragraphs you can say in fewer sentences.